### PR TITLE
Change licenses and update README.md

### LINKED ISSUE #1: Update readme
The readme should reflect more on the legal side and useability and not so much like a short abstract. Please refer to the example https://github.com/othneildrew/Best-README-Template/blob/master/BLANK_README.md

### LINKED ISSUE #2: Add an CC license for non code parts of the page
Code and Text Licenses should have different license types.

My suggestion:
- code: MIT
- text/images CC-BY-4.0

### PR DESCRIPTION
**Pull request by btschwertfeger**
* removed MIT license
* added  GNU GPLv3 for code
* added CC-BY-4.0 for text/images
* adjusted README.md 

Closes #1 
Closes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,14 +35,22 @@ This website was developed as part of [Benjamin T. Schwertfeger](https://b-schwe
 
 ---
 
-## Availability
+## 📍 Availability
+
+This website is available at the website of the Alfred-Wegener-Insitute (see [Growth_Model.html](https://www.awi.de/fileadmin/user_upload/AWI/Forschung/Klimawissenschaft/Dynamik_des_Palaeoklimas/Growth_Model)). The latest version is available the github pages feature of thie repositoriy (see [github.io/Growth-Model-Website/](https://btschwertfeger.github.io/Growth-Model-Website)).
```
Comment (tfoerst3r) [inline README.md]:
```suggestion
This website is available at [the website of the Alfred-Wegener-Insitute](https://www.awi.de/fileadmin/user_upload/AWI/Forschung/Klimawissenschaft/Dynamik_des_Palaeoklimas/Growth_Model). The latest version is available [the GitHub pages feature of the repository](https://btschwertfeger.github.io/Growth-Model-Website).
```

Comment (tfoerst3r) [reply README.md]: Looks good to me. Only in line 40 are some minor typos and maybe just pure linking. But this is a matter of taste

**Comment by tfoerst3r** [comment]
@btschwertfeger please have a look. Thanks for your distribution. Looks much better now.

**Comment by btschwertfeger** [comment]
@tfoerst3r looks fine to me. Let's wait for @n-sokolova - she wrote that she will have a look at this.

**Pull request merged**